spring-cloud-task-samples/multiple-datasources/src/main/java/io/spring/configuration/ExternalDataSourceConfiguration.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018-2019 the original author or authors.
```

Comment:
The year range should be `2018-2020` (same for other files).

Reply:
2020 already? :-D   Thanks for catching